index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Projects</title>
+
+    <style>
+      body {
+        padding: 0;
+        font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+      }
+
+      .flexbox-row {
+        display: flex;
+        gap: 10px;
+        flex-wrap: wrap;
+      }
+
+      .flexbox-col {
+        display: flex;
+        flex-direction: column;
+        padding-left: 10px;
+      }
+
+      .content-center{
+        justify-content: center;
+      }
+
+      .items-center{
+        align-items: center;
+      }
+
+      .button {
+        padding: 4px 8px;
+        box-shadow: 0 5px rgb(99, 98, 98);
+        background-color: rgb(155, 155, 155);
+        border-radius: 10px;
+      } 
+
+      .button:hover {
+        margin-top: 2px;
+        cursor: pointer;
+        box-shadow: 0 3px rgb(99, 98, 98);
+        margin-bottom: -2px;
+      }
+
+      .button:active {
+        margin-top: 3px;
+        cursor: progress;
+        box-shadow: 0 2px rgb(99, 98, 98);
+        margin-bottom: -3px;
+      }
+
+      div a {
```

Comment:
This works in this case but might create issues if you expand this page in the future. Currently, this selector will style every `<a/>` in any `<div/>` which essentially means that almost every link you decided to add on this page will get styled with this selector.

To be safe, I recommend changing this to something like `.link-list a` and adding a `link-list` class to the div with the links below